src/CuArrays.jl

```diff
@@ -62,6 +62,7 @@ include("utils.jl")
 include("indexing.jl")
 include("broadcast.jl")
 include("matmul.jl")
+include("matinv.jl")
```

Comment:
`matinv.jl` is pretty specific, maybe this functionality should go in `linalg.jl`, together with `matmul.jl`?

Reply:
I thought `linalg.jl` is for overloading `LinearAlgebra` module. There is also a `matmul.jl` already existing. But I'm happy to move `matinv.jl` into `linalg.jl` - no problem.

Reply:
It is, but in Base \(::Matrix, ::Matrix) is implemented in the LinearAlgebra stdlib too, so I guess this makes sens?

Reply:
Oh I didn't release this! Let me move the code into `linalg.jl` then!